pkgs/games/dwarf-fortress/dfhack/default.nix

```diff
@@ -33,40 +34,47 @@ let
     fi
   '';
 
-in stdenv.mkDerivation rec {
-  name = "dfhack-${version}";
+  dfhack = stdenv.mkDerivation rec {
```

Comment:
Okay this might be better to leave dfhack as is & create a "wrapper.nix" file to contain the buildEnv stuff.

Reply:
You're the guy with commit access, but IMO these two derivations are logically grouped, so it should be fine to keep them in the same file. Changes to one may well require changes to the other.

Reply:
It should be fine to merge, but won't work well until I can rework it based on the other one. I'll leave that decision up to you.